### ISSUE TITLE
Killometer problem

### ISSUE THREAD

**Issue by JigyasuSingh** [nic-delhi/AarogyaSetu_Android#337]
You will see that the location you are showing 1km  2km 5km 10km in my city I have corona postive that is 2km away from me but it's showing 10km

**Comment by Pranay1004** [comment]
How can you say this? I intend that you're not speaking something worth consideration? And anyways details of patients are not available on the app!

**Comment by JigyasuSingh** [comment]
Yes I agreed that but number of patience it's show and according to my City
news the Corona positive is away from me in less than 2km but it's showing
that postive is away from me 10km

**Comment by Pranay1004** [comment]
It is highly possible due to negligence maintained by the authorities and not the technical team. So be assured and believe that it is not updated by the people responsible!
